### PR TITLE
post checklist dashboard with no data

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
View for the teacher dashboard when the teacher has completed all the checklist objectives, but there is no data yet.

## WHY
This is the second to last major part of the teacher dashboard project. I initially intended to do the empty state and the populated version of this page together, but since the diff was already really large I thought it would be easier to review split this way.

## HOW
All new CSS and React code.

### Screenshots
Screen Shot 2021-04-09 at 10 43 42 AM
Screen Shot 2021-04-09 at 10 44 16 AM
Screen Shot 2021-04-09 at 10 44 32 AM
Screen Shot 2021-04-09 at 10 44 54 AM

### Notion Card Links
https://www.notion.so/quill/Home-Post-checklist-c6fa0a95ec1b4c8c92b8eac740e2bf9b

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | Not yet - this is all going to have to get shipped together, so I'm waiting til I'm done all the pieces to push it and get revisions
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Design Review: If applicable, have you compared the coded design to the mockups? | YES

### CODE REVIEW

**Review by cissyxyu — APPROVED** [review]
This is looking really good! Great work.

File: services/QuillLMS/client/app/bundles/Teacher/components/dashboard/activity_feed.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { requestGet } from '../../../../modules/request';
+
+const listIllustrationSrc = `${process.env.CDN_URL}/images/pages/dashboard/illustrations-list.svg`
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/dashboard/activity_feed.tsx:5]: think we want this with the rest in the Shared directory

File: services/QuillLMS/client/app/bundles/Teacher/components/dashboard/daily_tiny_tip.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+
+const baseImageUrl = `${process.env.CDN_URL}/images/pages/dashboard/daily_tiny_tips`
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/dashboard/daily_tiny_tip.tsx:3]: good call on pulling this out into its own variable

**Review by eadams17 — APPROVED** [review]
looks great! nice work 🎉

File: services/QuillLMS/client/app/bundles/Teacher/components/dashboard/activity_feed.tsx (hunk 1 above)
Comment (emilia-friedberg) [reply services/QuillLMS/client/app/bundles/Teacher/components/dashboard/activity_feed.tsx:5]: I have only been putting icons in the shared directory - I think for more specific illustrations like this, that are unlikely to be shared, it makes sense to keep those in the file where they're being used so it doesn't get too cluttered.

**Pull request merged**